docs/reasons-for-using-jekyll.md

```diff
@@ -0,0 +1,39 @@
+<h1>Reasons for using Jekyll</h1>
+
+Jekyll is a popular, free and open source static website generator which works like a content management system (for example, Drupal and WordPress) but unlike those systems, Jekyll generates at build-time rather than at request-time. Jekyll creates static websites with some templating abilities. 
+
+## Advantages of Jekyll 
```

Comment:
Since all of these benefits would apply to any static site, maybe change this heading to "Advantages of static sites"?

Also perhaps a quick introduction like: "Because it is built with Jekyll, the Open SDG platform gets all the advantages of using a static site."